packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -244,30 +245,36 @@ function FontCollection( { slug } ) {
 		return sortFontFaces( fontFamily.fontFace );
 	};
 
-	if ( renderConfirmDialog ) {
-		return <GoogleFontsConfirmDialog />;
-	}
-
-	const ActionsComponent = () => {
-		if ( slug !== 'google-fonts' || renderConfirmDialog || selectedFont ) {
-			return null;
-		}
+	if ( selectedCollection?.permission && ! hasPermission ) {
 		return (
-			<DropdownMenu
-				icon={ moreVertical }
-				label={ __( 'Actions' ) }
-				popoverProps={ {
-					position: 'bottom left',
-				} }
-				controls={ [
-					{
-						title: __( 'Revoke access to Google Fonts' ),
-						onClick: revokeAccess,
-					},
-				] }
-			/>
+			<div className="font-library__google-fonts-confirm">
+				<Card>
+					<CardBody>
+						<Text as="h3">
+							{ sprintf(
+								// translators: %s: Name of the Font Collection.
+								_x( 'Connect to %s' ),
+								selectedCollection.name
+							) }
+						</Text>
+						<Spacer margin={ 6 } />
```

Comment:
Not specific to this PR, but across the Font Library modal, I think it's better not to use Spacer components in this kind of implementation. This is because the component itself has zero height, its actual height is unpredictable, and the margin overlaps the elements before and after the spacer.

![image](https://github.com/WordPress/gutenberg/assets/54422211/925c31e3-b185-483e-a211-188b7ddeb6df)

We should be able to achieve the same layout by wrapping the elements directly under the `CardBody` component using a `VStack` component.